packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx

```diff
@@ -1,13 +1,41 @@
 import React, { useState } from 'react';
 import { LoadingOverlay, Switch } from '@mantine/core';
-import styled from 'styled-components';
+import styled from '@emotion/styled';
+import { css, cx } from '@emotion/css';
 import chroma from 'chroma-js';
+
 import { useNovuTheme } from '../../../../hooks';
 import { getChannel } from './channels';
 import { switchStyles, Text } from './styles';
 import { getLinearGradientColorStopValues } from '../../../../shared/utils/getLinearGradientColorStopValues';
 import { Check } from '../../../../shared/icons/Check';
 import { colors } from '../../../../shared/config/colors';
+import { useStyles } from '../../../../store/styles';
+
+const iconClassName = (
+  active: boolean,
+  iconColor: {
+    active?: string;
+    inactive?: string;
+  }
+) => css`
+  color: ${active ? iconColor.active : iconColor.inactive};
+  width: 30px;
+`;
+
+const channelLabelClassName = (
+  active: boolean,
+  fontColor: {
+    active?: string;
+    inactive?: string;
+  }
+) => css`
+  color: ${active ? fontColor.active : fontColor.inactive};
```

Comment:
This can fail if both are properties are not set. What if we prevent it like this?
```suggestion
  color: ${fontColor?.active ?? fontColor?.inactive ?? 'a-default-value-here'};
```
About setting a potential default is something we can do with the style changes you have done?

Reply:
I've just moved this code from: `<Text size={'md'} color={active ? fontColor.active : fontColor.inactive}>`

From what I do see we have a default theme config objects where these are defined, then they are merged with what the user does provide, so there never would be the case when none of them are set. But I don't know why the types were defined like that before.

Reply:
If you checked no bug will happen, all good.